docs/developer-guide/mapstore-migration-guide.md

```diff
@@ -22,6 +22,37 @@ This is a list of things to check if you want to update from a previous version
 
 ## Migration from 2023.02.xx to 2024.01.00
 
+### Added possibility to create a fine grid for widgets also in the main viewer page
+
+There a new cfg object called *defaults* that contains for now only mdSize where you can put the custom height and width when creating a new widget.
+
+For example use the following in the plugins.desktop array of `localConfig.json`:
+
+```json
+    {
+        "name": "Widgets",
+        "cfg": {
+            "rowHeight": 10,
+            "cols": {
+                "md": 48,
+                "xxs": 1
+            },
+            "defaults": {
+                "mdSize": {
+                    "h": 8,
+                    "w": 8
+                }
+            }
+        }
+    }
+```
+
+**CAVEAT:**
+
+```txt
+Keep in mind that old widgets will shrink due to the new cols value, so you have to manually edit those maps to restore correct size
+```
+
```

Comment:
> This is not needed. You don't have to apply any of these changes to migrate to a new version, so it is not a part of the migration guideline.

well it's not mandatory it's informative. i think it could be good to better expose these kind of enhancements, but I see your point